src/assets/carousel.css

```diff
@@ -0,0 +1,93 @@
+/* GLOBAL STYLES
+-------------------------------------------------- */
+/* Padding below the footer and lighter body text */
+
+body {
+  padding-top: 3rem;
+  padding-bottom: 3rem;
+  color: #5a5a5a;
+}
+
```

Comment:
Noted. I just used carousel.css since its only used for the sample page(that might be only temporary). But yh sure, I'll the put classes where necessary